app/logic/daily_digest.rb

```diff
@@ -10,22 +10,25 @@ def perform
   private
 
   def get_statistic
-    recipients = get_recipients
-    completed = completed_amount
-    remaining = incompleted_amount
-
-    { recipients: recipients, completed: completed, remaining: remaining }
+    completed = completed_for_today
+
+    {
+      recipients: get_recipients,
+      completed_todos: completed.group_by { |item| item.list },
+      completed_amount: completed.size,
+      remaining_amount: incompleted
+    }
   end
 
   def get_recipients
     User.all.pluck :email
   end
 
-  def completed_amount
-    Item.completed_today.count
+  def completed_for_today
+    Item.with_list.completed_today
   end
 
-  def incompleted_amount
+  def incompleted
     Item.incompleted.count
   end
```

Comment:
@vlasiak I suggest to revert renaming back, as `incompleted` sounds like it returns incompleted objects instead of their amount.